src/k8s-engine/k3sHelper.ts

```diff
@@ -503,6 +506,24 @@ export default class K3sHelper extends events.EventEmitter implements VersionLis
         writeStream.end(userYAML, 'utf-8');
       });
       await fs.promises.rename(workPath, userPath);
+
+      // The config file we modified might not be the top level one.
+      // Update the current context.
+      console.log('Setting default context...');
+      await new Promise<void>((resolve, reject) => {
+        const child = childProcess.spawn(
+          resources.executable('bin/kubectl'),
```

Comment:
The `bin/` part can get dropped once the two issue #155 branches are merged to main.

Reply:
Fixed in 09ddd67bf35d0c93db9ee7fb8f7796204846478d